ToDo/ToDo/ViewController.swift

```diff
@@ -60,11 +89,12 @@ extension ViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView,
                    cellForRowAt indexPath: IndexPath)
         -> UITableViewCell {
```

Comment:
🍰I AM CAKING THIS UNDER DURESS. But looking at it makes me itch. You do you, none of this is wrong. It's just _gross_ 🤣

Reply:
🍰 style changes made! 🎉